doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -105,3 +134,19 @@ The behavior of the implementation is almost the same as Zephyr's with the follo
   * All ``flags`` are sent to the network core when either the :c:func:`bt_gatt_subscribe` or :c:func:`bt_gatt_resubscribe` function is called.
     This covers most of the cases, because the ``flags`` are normally set once before those functions calls.
   * If you want to read or write the ``flags`` after the subscription, you have to call :c:func:`bt_rpc_gatt_subscribe_flag_set`, :c:func:`bt_rpc_gatt_subscribe_flag_clear` or :c:func:`bt_rpc_gatt_subscribe_flag_get`.
```

Comment:
Another comment based on the AI feedback _(Are there any specific limitations or considerations when using the Bluetooth RPC library with devices that have more complex Bluetooth Low Energy requirements, such as multiple simultaneous connections or advanced security features?):_ can we add another bullet or a note saying something like "using advanced Bluetooth LE configurations, such as multiple simultaneous connections or advanced security features can be a limitation as the child image (:ref:`ble_rpc_host` or :ref:`ipc_radio`) may require significantly more memory that may not be available on the MCU it runs on. Typically, network or Radio cores are more memory-constrained than the application MCU"